server/controllers/notifications.js

```diff
@@ -0,0 +1,77 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+//get all notifications
+exports.getAllNotifications = asyncHandler(async (req, res, next) => {
```

Comment:
Can you run prettier on this file, looks like the spacing and semicolons are a little off